integration/user_api_test.go

```diff
@@ -869,7 +988,7 @@ func (t *testEmailer) SendInviteEmail(email string, redirectURL url.URL, clientI
 	return retURL, nil
 }
 
-func makeUserToken(issuerURL url.URL, userID, clientID string, expires time.Duration, privKey *key.PrivateKey) string {
+func makeToken(issuerURL url.URL, userID, clientID string, expires time.Duration, privKey *key.PrivateKey) string {
```

Comment:
ah my bad, I see why you did this. perhaps create a different method that doesn't take a clientID instead?